actioncable/lib/action_cable/connection/subscriptions.rb

```diff
@@ -42,6 +42,8 @@ def remove(data)
 
       def remove_subscription(subscription)
         subscription.unsubscribe_from_channel
+      ensure
```

Comment:
Only ensures the subscription is removed and not that channels are unsubscribed from.

Depending on how exceptions are swallowed, we might need to run the unsubscribe outside of the unsubscribe callbacks. But that doesn't sound great.